### PR TITLE
python-dbus: rebottle

### PR DESCRIPTION
**Pull request by danielnachun**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/linuxbrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/linuxbrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?
- [ ] Have you included the output of `brew gist-logs <formula>` of the build failure if your PR fixes a build failure. Please quote the exact error message.

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Why rebottle?

**Comment by danielnachun** [comment]
I forgot to write the reason.  The bottles fails its test because it was built with `python@3.7`, but we are now up to `python@3.9`.  When I rebuild locally in Docker, everything is fine.  So I assume the bottle is just old?

**Comment by iMichka** [comment]
I also see the error:
```
==> Installing python-dbus
==> Pouring python-dbus--1.2.14.x86_64_linux.bottle.tar.gz
Warning: Packages have been installed for:
  Python 3.7
but this formula depends on:
  Python 3.9
==> Summary
🍺  /home/linuxbrew/.linuxbrew/Cellar/python-dbus/1.2.14: 29 files, 489.0KB
```

As the bottle already exists, I would rather revision bump this. You can now completely delete bottle block, I fixed the bug that prevented to create a new bottle block.

File: Formula/python-dbus.rb [hunk 1]
```diff
@@ -3,10 +3,7 @@ class PythonDbus < Formula
   homepage "https://dbus.freedesktop.org/doc/dbus-python/"
   url "https://dbus.freedesktop.org/releases/dbus-python/dbus-python-1.2.14.tar.gz"
   sha256 "b10206ba3dd641e4e46411ab91471c88e0eec1749860e4285193ee68df84ac31"
-
-  bottle do
-    sha256 cellar: :any_skip_relocation, x86_64_linux: "c20d0729e24a65d7f202377775e92401df849d1bb203785b688e57263aaaba4a"
-  end
+  revision 1
```
Comment (iMichka) [inline Formula/python-dbus.rb:6]: revision 1 unless OS.mac?

Comment (danielnachun) [reply Formula/python-dbus.rb:6]: Is this needed if the formula has `depend_on :linux`?  On that note, I'm not actually sure this really is a Linux-only formula.

Comment (iMichka) [reply Formula/python-dbus.rb:6]: Right, I missed that.

I'm wondering why we have this formula at all. It's used nowhere and the install count is very low.

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**